src/main/scala/org/broadinstitute/dsde/workbench/sam/config/GoogleServicesConfig.scala

```diff
@@ -31,6 +31,12 @@ final case class GoogleServicesConfig(
     groupSyncTopic: String,
     groupSyncSubscription: String,
     groupSyncWorkerCount: Int,
```

Comment:
Could we contain all of these in a `PubSubConfig` case class like in the other PR? Otherwise, I think I like this approach more

Reply:
Absolutely. Although I think that could be its own independent PR.